mirdata/utils.py

```diff
@@ -29,11 +29,45 @@
 import hashlib
 import os
 import json
+import types
 
 
 MIR_DATASETS_DIR = os.path.join(os.getenv('HOME', '/tmp'), 'mir_datasets')
 
 
+class Track(object):
```

Comment:
i might have put this in `track.py`. `utils.py` is a bit of an antipattern, or at least good to avoid.

Reply:
good idea - moving!